### ISSUE TITLE
Low Acc when training ssm_2d

### ISSUE THREAD

**Issue by anhtienng** [ethanbar11/ssm_2d#4]
Hi,

I am using your existing code and setting to run ssm_2d on CIFAR-10,
but the top-1 accuracy is low.

`best top-1: 57.20, final top-1: 56.30`

I'm running `python main.py --model vit --dataset CIFAR10 --emb ssm_2d`.
Are there any configs that I should change ?

Thank you.

**Comment by ethanbar11** [comment]
We didn't test it on CIFAR-10 (the results aren't shown in the paper) and I don't remember the hyper-parameters you need to use. I'd suggest starting from one of the other ones, CIFAR-100, Tiny-ImageNet or ImageNet
